lib/core/src/lib/dynamic-chip-list/adf-strings-to-chips.pipe.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * @license
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+import { Chip } from './chip';
+
+@Pipe({
+    name: 'adfStringsToChips',
+    standalone: true
+})
+export class AdfStringsToChipsPipe implements PipeTransform {
```

Comment:
we don't need this pipe to be in the core, please just use this code in the component instead

Reply:
transforming other way will need much more problematic changes, that was the cleanest way to do it

Reply:
you can do the same in the component, what's the problem moving this function there? this pipe does zero performance saving as it's not even marked as pure

Reply:
Pipes are pure by default, so no need mark it manually. Also `map` function is returning brand new array.

In this scenario I am not changing any variable data type, so the change happens while passing data between components. There is already a lot of different variables for different operations.

The only way is to add input field for string array, which will need additional checks on every array change, which will cause additional calls on change.